css/excercise_one/part-c/scss/_variables.scss

```diff
@@ -0,0 +1,59 @@
+$default-font-family: 'Poppins', sans-serif;
+$primary-color: black;
+$secondary-color: rgba(15, 88, 183, 1);
+$tertiary-color: rgba(155, 199, 255, 1);
+$background-color: white;
+$secondary-background-color: rgba(242, 247, 255, 1);
+$tertiary-background-color: rgba(15, 88, 183, 1);
```

Comment:
I would remove the `background` from variable names, we usually just set a color's name for example primary `white` or `success` and then we set classes such as `bg-primary` or `bg-white` instead of having color's name specifically imply that they are for background (because you could use them for something other than a background too)